src/applications/vaos/referral-appointments/tests/utils/provider.unit.spec.js

```diff
@@ -10,9 +10,9 @@ describe('VAOS provider utils', () => {
   });
   const tomorrow = addDays(startOfDay(new Date()), 1);
   describe('createProviderDetails', () => {
-    const providerObjectTwoSlots = providerUtil.createProviderDetails(2, '222');
+    const providerObjectTwoSlots = providerUtil.createProviderDetails(3, '222');
```

Comment:
This test was failing due to the `provider` objects being persisted and resulting in wrong assertions due to re-used references

Reply:
I have a fix for this in my PR that is out

Reply:
Used the fix from your PR and it seemed to work